src/lib/operators/join_hash/join_hash_steps.hpp

```diff
@@ -182,34 +183,71 @@ class PosHashTable {
   std::optional<std::vector<std::pair<HashedType, Offset>>> _values{std::nullopt};
 };
 
+// The bloom filter (with k=1) is used during the materialization and build phases. It contains `true` for each
+// `hash_function(value) % BLOOM_FILTER_SIZE`. Much of this could be improved, for example by (1) dynamically checking
+// whether bloom filters make sense, (2) choosing an appropriate filter size, and (3) working with multiple hash
+// functions.
+static constexpr auto BLOOM_FILTER_BITS = 20;
```

Comment:
It's not really the number of bits, right? That's `BLOOM_FILTER_SIZE`.

Reply:
Not sure if I can follow. The bloom filter's size is `1 << 20 == 2^20`.

Reply:
Yes, the size is is 2^20 and a bloom filter consists of bits, so the number of bits is 2^20.
And then there is a variable named `_BITS` that is just 20. That's confusing.

Reply:
Honestly, we don't need `BLOOM_FILTER_BITS` anyway.

```
static constexpr auto BLOOM_FILTER_SIZE = 1 << 20;
static constexpr auto BLOOM_FILTER_MASK = BLOOM_FILTER_SIZE - 1;
```